apps/site/src/components/pages/wordpress/header.tsx

```diff
@@ -1,16 +1,18 @@
-import { Stream } from "@cloudflare/stream-react";
-import { Box, Container, Skeleton, Typography, useTheme } from "@mui/material";
-import Image from "next/legacy/image";
-import { useState } from "react";
+import { Box, Container, Typography, useTheme } from "@mui/material";
 
-import wpHelixImage from "../../../../public/assets/new-home/wp-helix.png";
 import { BpWpIcons } from "./bp-wp-icons";
-import { EarlyAccessCTA } from "./early-access-cta";
+import { CustomButton } from "./custom-button";
 
 export const Header = () => {
   const theme = useTheme();
 
-  const [loading, setLoading] = useState(true);
+  const onInstallWordPress = () => {
+    // TODO
+  };
+
+  const onDownload = () => {
+    // TODO
+  };
```

Comment:
@nonparibus, These functions as the name describes are meant to be called when the individual buttons are clicked, a `// TODO` is placed because we do not know the functionality details and how you'd want it implemented